lib/manager/components/version/VersionSelectorDropdown.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import * as React from 'react'
+import { Dropdown, MenuItem } from 'react-bootstrap'
+
+import VersionRetrievalBadge from './VersionRetrievalBadge'
+
+import type { FeedVersion } from '../../../types'
+
+type Props = {
+  dropdownProps: any,
+  header?: string,
+  itemFormatter: (FeedVersion, number, FeedVersion) => React.Node,
+  title: React.Node,
+  version: FeedVersion,
+  versions: ?Array<FeedVersion>
+}
+
+const defaultItemFormatter = (version: FeedVersion, index: number, activeVersion: FeedVersion) => {
+  return (
+    <MenuItem
+      key={version.id}
+      eventKey={version.version}
+    >
+      {version.id === activeVersion.id
+        ? <Icon
+          style={{
+            position: 'absolute',
+            left: '2px',
+            marginTop: '3px'
+          }}
+          type='check' />
+        : null}
+      {version.version}. {version.name}{' '}
+      <VersionRetrievalBadge version={version} />
+    </MenuItem>
+  )
+}
+
+export default class VersionSelectorDropdown extends React.Component<Props> {
+  static defaultProps = {
+    dropdownProps: {
+      onSelect: (index: number) => console.log(`selected ${index}`)
+    },
+    itemFormatter: defaultItemFormatter
+  }
+
+  render () {
+    const {
+      dropdownProps,
+      header,
+      itemFormatter,
+      title,
+      version,
```

Comment:
Suggestion: also rename `version` to `activeVersion`.

Reply:
Leaving it named version conforms with the name of the prop in other version components. I do somewhat agree with you on this item, but I think it should be handled in a separate PR and deserves a little more consideration on whether we should make the change.